### DISCUSSION TITLE
Dividers option meeaning

### DISCUSSION THREAD

**Discussion by stockiNail** [kurkle/chartjs-chart-treemap#170] [Q&A] [answered]
Having a look to the `dividers` option, it is not clear to me if the current bahvior is the wished one or not.

At the moment, setting the dividers option, the box is splitted by 2, ignoring the amount of subelements.

In the sample (https://chartjs-chart-treemap.pages.dev/samples/dividers.html), the passed data are the following:

```javascript
const data = [
  {category: 'main', value: 1},
  {category: 'main', value: 2},
  {category: 'main', value: 3},
  {category: 'other', value: 4},
  {category: 'other', value: 5},
];
```

and my expectation, maybe wrong, grouping by category, is to have 2 boxes with 2 dividers for main and 1 for other. Instead, there is 1 divider for main and 1 for other.

**Comment by kurkle** [comment, marked as answer]
Looks wrong indeed. As I'm probably the only one ever used that option and my use case was a one time print, maybe the feature should just be removed.

**Reply by stockiNail** [reply]
I agree! Let me copy the discussion to an issue
